### ISSUE TITLE
Adding additional command line tools for workshops

### ISSUE THREAD

**Issue by utherp0** [openshift-labs/workshop-spawner#27]
Would it be possible to add the following CLIs to the workshop image - this is for OCP4.2+ workshops:

odo
tkn (or whatever the Tekton CLI now is called)
kamel (for camel-k operator interactions)

**Comment by GrahamDumpleton** [comment]
This repository is only the spawner. The separate terminal image which the tools are installed in, and which the workshop dashboard image is derived from is:

* https://github.com/openshift-homeroom/workshop-terminal

It already includes `odo` as well as `kubectl` and `oc`.

As to `tkn` or other tools, the idea is that you would include what you need in the workshop image you created which derives from the dashboard image.

You can see how this is done in the existing tekton pipelines workshop in:

* https://github.com/openshift-labs/lab-tekton-pipelines/blob/master/.workshop/build

In other words, you build it in the workshop image which requires it.

Because there are too many such tools people may want and different workshops may feasibly need different versions of the tools to match the version the workshop is written for, it isn't practical to include them in the terminal base image.

**Comment by GrahamDumpleton** [comment]
Actually, should point out that where that `build` script installs `tkn` is actually wrong. It shouldn't be using its own `.bin` directory. Any binaries should be installed in `/opt/app-root/bin`. I will need to get that fixed.

**Comment by GrahamDumpleton** [comment]
Hmm, I actually already did raise the problem, but it still hasn't been changed.

* https://github.com/openshift-labs/lab-tekton-pipelines/issues/19
